### ISSUE TITLE
A11y: Re-enable Automated Scrolling with A11y warning for Product Sliders 

### ISSUE THREAD

**Issue by shopwareBot** [shopware/platform#7623]
Currently, the automated scrolling control is completely disabled. We will refactor the control so that it is:

* disabled by default
* can be re-enabled, but will require an acknowledgement that doing so will cause accessibility issues

**Comment by shopwareBot** [comment]
[public] Shopware automated response: This issue is linked to the internal issue: **NEXT-40947**



[created from NEXT-40947, comment 545453]

**Comment by marcelbrode** [comment]
Warning not visible because

https://github.com/shopware/meteor/issues/615
